src/Login/Login.test.js

```diff
@@ -6,43 +6,39 @@ import Login from './Login';
 import { MemoryRouter } from 'react-router-dom';
 
 describe('Login', () => {
-//Tests:
-    //handleInputChange(event)
-    //handleLogin is fired (userEvent) when 
-        //submit button is clicked
-    //can we/should we test componentDidMount?
+
     it('should render a login form', () => {
-        // const value1 = 'charlie@turing.io'
-        // const value2 = 'qwerty'
         const fakeUpdateLogin = jest.fn();
-        const fakeHandleChange = jest.fn();
-        const handleLogin = jest.fn();
-        const myUser = {}; 
+        const fakeSetUser = jest.fn();
+        const myUser = {};
 
       render(
         <MemoryRouter>
-          <Login 
+          <Login
             user={myUser}
             updateLoginView={fakeUpdateLogin}
-            // handleInputChange={fakeHandleChange}
+            setUser={fakeSetUser}
           />
-        </MemoryRouter> 
+        </MemoryRouter>
       )
 
+      expect(fakeUpdateLogin).toHaveBeenCalledTimes(1);
+
       userEvent.type(screen.getByPlaceholderText('example@email.com'), 'charlie@turing.io');
       userEvent.type(screen.getByPlaceholderText('enter password'), 'qwerty');
-      
+
       expect(screen.getByPlaceholderText('example@email.com')).toBeInTheDocument();
       expect(screen.getByPlaceholderText('enter password')).toBeInTheDocument();
       expect(screen.getByPlaceholderText('example@email.com')).toHaveValue('charlie@turing.io');
       expect(screen.getByPlaceholderText('enter password')).toHaveValue('qwerty');
       expect(screen.getByRole('button', { name: 'Login' })).toBeInTheDocument();
-      
+
       userEvent.click(screen.getByText('Login'));
```

Comment:
We could split these assertions into two unit tests: 
One that is responsible for:
  + checking that the inputs fields with the placeholder text are in the document
  +  `userEvent.type( )` in both inputs
  + checking that the inputs field have the expected value after the 'user' has typed

Another that is responsible for:
  + checking that the button exists in the document
  + `userEvent.click( )` occurs on the button (after typing)
  + checking that both inputs have empty values after the click has occurred (expect change in `Login` view

We could also split this up differently or even more granularly.